kt/plugins/godot-intellij-plugin/build.gradle.kts

```diff
@@ -0,0 +1,98 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import org.jetbrains.changelog.closure
+import org.jetbrains.changelog.markdownToHTML
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    // Java support
+    id("java")
+    // Kotlin support
+    id("org.jetbrains.kotlin.jvm")
+    // gradle-intellij-plugin - read more: https://github.com/JetBrains/gradle-intellij-plugin
+    id("org.jetbrains.intellij") version "0.6.5"
+    // gradle-changelog-plugin - read more: https://github.com/JetBrains/gradle-changelog-plugin
+    id("org.jetbrains.changelog") version "0.6.2"
+    // detekt linter - read more: https://detekt.github.io/detekt/gradle.html
+    id("io.gitlab.arturbosch.detekt") version "1.14.2"
+    // ktlint linter - read more: https://github.com/JLLeitschuh/ktlint-gradle
+    id("org.jlleitschuh.gradle.ktlint") version "9.4.1"
+}
+
+group = "com.utopia-rise"
+version = "0.0.1"
+
+dependencies {
+    implementation(project(":godot-library"))
+    detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.14.2")
+}
+
+// Configure gradle-intellij-plugin plugin.
+// Read more: https://github.com/JetBrains/gradle-intellij-plugin
+intellij {
```

Comment:
If we are planning to allow this plugin to be used in android studio at some point, then I would suggest to have a release artifact per minor version of IJ. See https://github.com/spekframework/spek/blob/2.x/spek-ide-plugin-intellij-idea/build.gradle.kts#L42.